build.gradle

```diff
@@ -315,8 +315,8 @@ signing {
 
 
 //signing {
-//    def signingKey = findProject("signingKey")
-//    def signingPassword = findProject("signingPassword")
+//    def signingKey = "$System.env.SIGNING_KEY"
```

Comment:
and maybe we should rename the ENV variable names to something that won't collide with an users's existing "SIGNING_KEY" for something else. Perhaps: `FW_JLIB_SIGNING_KEY` and `FW_JLIB_SIGNING_PASSWORD`